apiary.apib

```diff
@@ -1071,14 +1071,20 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
         [
             {
+                "description": "Worked for lots of successful companies.",
```

Comment:
When I tested this in apiary's text editor, there are some _semantic_ issues.

Can you take a quick read through:  https://github.com/OperationCode/operationcode_backend#api-documentation

And see if you can resolve them?